### PR TITLE
Updating sample invocation of getLaunchIntent()

### PR DESCRIPTION
**Pull request by kmayoral**
Updated `Chucker.getLaunchIntent(context)` to `Chucker.getLaunchIntent(context, Chucker.SCREEN_HTTP)`

## :camera: Screeshots
*Show us what you've changed, we love images.*

## :page_facing_up: Context
*Why did you changed something? Is there an [issue](https://github.com/ChuckerTeam/chucker/issues) to link here? Or an extarnal link?*

## :pencil: Changes
*Which code did you change? How?*

## :paperclip: Related PR
*PR that blocks this one, or the ones blocked by this PR*

## :warning: Breaking
*Is there something breaking in the API? Any class or method signature changed?*

## :pencil: How to test
*Is there a special case to test your changes?*

## :crystal_ball: Next steps
*Do we have to plan something else after the merge?*

### CODE REVIEW

**Comment by cortinico** [comment]
Thanks for the PR and thanks for spotting the problem.
Looks like we missed this API change in the migrating document :(

That being said, ideally we should change to hide the paratemers list. So something like:

```
So if to launch the UI of Chucker, you would normally call:

Chuck.getLaunchIntent(...)

now you will call

Chucker.getLaunchIntent(...)
```

This will not confuse the reader.
Moreover can I ask you to fix the same problem also in the 
`docs/migrating-from-chuck.md` file?

File: docs/migrating-from-2.0.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Chuck.getLaunchIntent(context)
 now you will call
 
 ```kotlin
-Chucker.getLaunchIntent(context)
+Chucker.getLaunchIntent(context, Chucker.SCREEN_HTTP)
```
Comment (cortinico) [inline docs/migrating-from-2.0.md]:
```suggestion
Chucker.getLaunchIntent(...)
```

Comment (olivierperez) [reply docs/migrating-from-2.0.md]: :+1:

File: docs/migrating-from-2.0.md [hunk 2]
```diff
@@ -92,4 +92,4 @@ The `Period` enum has also been moved from `ChuckCollector` to `RetentionManager
 
 ## 5. registerDefaultCrashHanlder typo
 
-The function `Chuck.registerDefaultCrashHanlder` contained a typo in the name and now is moved to `Chucker.registerDefaultCrashHandler`.
\ No newline at end of file
+The function `Chuck.registerDefaultCrashHanlder` contained a typo in the name and now is moved to `Chucker.registerDefaultCrashHandler`.
```
Comment (olivierperez) [inline docs/migrating-from-2.0.md]: What's the change here?

Comment (cortinico) [reply docs/migrating-from-2.0.md]: New like at the end of file change, so we can ignore this.

**Pull request merged**